input/fsh/modules/family.fsh

```diff
@@ -0,0 +1,175 @@
+* $snomedct_us#415684004  "Suspected"
+* $snomedct_us#410515003  "Known present"
+* $snomedct_us#261665006  "Unknown"
+
+Extension: Consanguinity
+Id: consanguinity
+Title: "Consanguinity Extension"
+Description: "Extension containing Consanguinity"
+* value[x] only CodeableConcept
+* valueCodeableConcept from consanguinity-assertion-vs (extensible)
+* valueCodeableConcept ^short = "Is there known or suspected consanguinity in this study family?"
+
+Extension: StudyFamilyFocus
+Id: study-family-focus
+Title: "Study Family Focus Extension"
+Description: "Extension containing Study Family Focus"
+* value[x] only CodeableConcept 
+* valueCodeableConcept ^short = "What is this study family investigating? EG, a specific condition"
+
+Profile: NcpiStudyFamily
+Parent: Group
+Id: ncpi-study-family
+Title: "NCPI Study Family"
+Description: "Study Family"
+* ^version = "0.1.0"
+* ^status = #draft
+* id 1..1 
+* id ^short = "ParticipantID - Unique participant identifier. System identifier used for internal references."
+* identifier 0..* 
+* identifier ^short = "External IDs for this participant. Requires scoping."
+* extension contains FamilyType named family-type 0..1
+* extension[family-type] ^short = "Describes the 'type' of study family, eg, trio."
+* extension contains Description named description 0..1
+* extension[description] ^short = "Free text describing the study family, such as potential inheritance or details about consanguinity"
+* extension contains Consanguinity named consanguinity 0..1
+* extension[consanguinity] ^short = "Is there known or suspected consanguinity in this study family?"
+* extension contains StudyFamilyFocus named study-family-focus 0..1
+* extension[study-family-focus] ^short = "What is this study family investigating? EG, a specific condition"
+
+// Family Role
+
+Logical: CdmFamilyRole
+Id: SharedDataModelFamilyRole
+Title: "Shared Data Model for Family Role"
+Description: "The **Shared Data Model for Family Role**"
+* participant 1..1 Reference "The participant we are describing"
+* studyFamily 1..1 Reference "The study family this participant is part of"
+* familyRole 0..1 code "The role this individual has in the family, specifically with respect to a proband or index participant"
+
+Extension: StudyFamily
+Id: study-family
+Title: "Study Family Reference"
+Description: "Extension containing Study Family Reference"
+* insert SetContext(Group)
+* value[x] only Reference
+* valueReference 1..1
+* valueReference only Reference(NcpiStudyFamily)
+
+Extension: FamilyRole
+Id: family-role
+Title: "Study Family Focus"
+Description: "Extension containing Family Role"
+* value[x] only CodeableConcept 
+* valueCodeableConcept ^short = "The role this individual has in the family, specifically with respect to a proband or index participant"
+* valueCodeableConcept from $ncpi-family-member (extensible)
+
+Profile: NcpiFamilyRole
+Parent: Group
+Id: ncpi-family-role
+Title: "Shared Data Model for Family Role"
```

Comment:
These aren't part of the shared data model. I think you can just refer to them as "Family Role" or something like that. I suspect this was a copy paste error:)